### PR TITLE
📝 Change supported node versions

### PR DESCRIPTION
**Pull request by JackDevAU**
### General Contributing:

- [ ] Have you followed the guidelines in our [Contributing document](https://github.com/tinacms/tina.io/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### All New Content Submissions: (To be confirmed by reviewer)

- [ ] Title is short & specific
- [ ] Headers are logically ordered & consistent
- [ ] Purpose of document is explained in the first paragraph
- [ ] Procedures are tested and work
- [ ] Any technical concepts are explained or linked to
- [ ] Document follows structure from templates
- [ ] All links work
- [ ] The spelling and grammar checker has been run
- [ ] Graphics and images are clear and useful
- [ ] Any prerequisites and next steps are defined.

### CODE REVIEW

File: components/layout/setup-overview.tsx [hunk 1]
```diff
@@ -72,8 +72,8 @@ const OverviewTemplate = (props) => {
             <h1>Getting Started With Tina</h1>
             <hr />
             <blockquote>
-              TinaCMS supports the latest LTS version of Node. We currently
-              support v18, v20, and v22 of Node with NPM, PNPM, and Yarn.
+              TinaCMS requires Node version 18 or higher, including all LTS
+              releases, and supports NPM, PNPM, and Yarn.
```
Comment (wicksipedia) [inline components/layout/setup-overview.tsx:76]:
```suggestion
              TinaCMS is supported only on currently maintained LTS versions of Node.js and works with NPM, PNPM, and Yarn.
```

File: content/docs/introduction/faq.mdx [hunk 2]
```diff
@@ -17,7 +17,7 @@ For more information, see [TinaCloud Overview](/docs/tina-cloud/overview).
 
 ## What versions of Node are supported?
 
-TinaCMS supports the latest LTS version of Node. We currently support v18, v20, and v22 of Node with NPM, PNPM, and Yarn.
+TinaCMS requires Node version 18 or higher, including all LTS releases, and supports NPM, PNPM, and Yarn.
```
Comment (wicksipedia) [inline content/docs/introduction/faq.mdx:20]:
```suggestion
TinaCMS is supported on currently maintained LTS versions of Node.js and works with NPM, PNPM, and Yarn.
```

File: components/layout/setup-overview.tsx (hunk 1 above)
Comment (JackDevAU) [reply components/layout/setup-overview.tsx:76]: Is it worth stating the minimum version is 18 or do you think this covers it

Comment (wicksipedia) [reply components/layout/setup-overview.tsx:76]: 18 will not be supported after April so I'd avoid adding in the maintenance debt

**Review by wicksipedia — APPROVED** [review]

**Pull request merged**